Additions to codegen initiative (#22)
trifectatechfoundation/trifectatech-website: content/initiatives/codegen.md

### What's next

> The improvements the RFC gives are niche. What it does show is that there is room for improvement. While outperforming C by 5% might be unlikely, even a shift from -2% to +2% would significantly change Rust's story and, more importantly, its impact.

Reviewer: ```suggestion
The improvements of RFC 3720 are only relevant for specific programs, but it shows that there is room for improvement. While outperforming C by 5% might be unlikely, even a shift from -2% to +2% would significantly improve Rust's story and, more importantly, its impact.
```

> These improvements are achievable if we can better use Rust's advantages over C, i.e., having access to more information about how the compiler can optimize.

Reviewer: ```suggestion
The gap with C can be closed if we can better use Rust's advantages over C, i.e., having access to more information about how the compiler can optimize.
```

Reviewer: still not a great sentence, it doesn't say much.

Author: Agreed, but it does hint at (that we understand) why this is even feasible
